test/Serilog.Tests/Context/LogContextTests.cs

```diff
@@ -25,6 +26,51 @@ public LogContextTests()
         }
 
         [Fact]
+        public void PushedPropertiesAreAvailableToLoggers()
+        {
+            LogEvent lastEvent = null;
+
+            var log = new LoggerConfiguration()
+                .Enrich.FromLogContext()
+                .WriteTo.Sink(new DelegatingSink(e => lastEvent = e))
+                .CreateLogger();
+
+            using (LogContext.PushProperty("A", 1))
+            using (LogContext.Push(new PropertyEnricher("B", 2)))
+            using (LogContext.Push(new PropertyEnricher("C", 3), new PropertyEnricher("D", 4))) // Different overload
+            {
+                log.Write(Some.InformationEvent());
+                Assert.Equal(1, lastEvent.Properties["A"].LiteralValue());
+                Assert.Equal(2, lastEvent.Properties["B"].LiteralValue());
+                Assert.Equal(3, lastEvent.Properties["C"].LiteralValue());
+                Assert.Equal(4, lastEvent.Properties["D"].LiteralValue());
+            }
+        }
+
+        [Fact]
+        public void LogContextCanBeCloned()
```

Comment:
How does cloning work across threads and logical call contexts? Is there any sane way to test this?